docs/clusters/alpine/allocations.md

```diff
@@ -43,23 +43,25 @@ resources requested, job size, and QOS.
 ### Allocations
 When you receive a Research Computing account you are automatically 
 assigned a **Trailhead Auto-Allocation**, which grants you a fixed share 
-of `ucb-general`. The Trailhead is a great allocation for smaller jobs or 
+of `ucb-general`. RMACC users are granted a fixed share of `rmacc-general`. 
+The Trailhead is a great allocation for smaller jobs or 
 testing and benchmarking your code. However, if you consume more than your 
-fixed share of `ucb-general`,  your `LevelFS` will decrease, causing your 
+fixed share of `ucb-general` (or `rmacc-general`),  your `LevelFS` will decrease, causing your 
 priority score to decrease. The end result is that your jobs will sit 
-lower (read: longer) in the queue relative to other jobs. One way to 
+lower in the queue relative to other jobs. One way to 
 combat this is to apply for an allocation.
 
 ## Alpine Allocation Tiers
 
-In addition to the Trailhead auto-allocation (`ucb-general`) that all users are awarded automatically, CURC offers two 
```

Comment:
I think we can add a few sentences to this section that clarify the intended audience and direct AMC and CSU to who to contact about allocations. Suggest editing the first sentence "..., CURC offers CU Boulder users two additional tiers..." and somewhere in this section, maybe at the end, "CSU and AMC users who are interested in learning about their institution's Alpine allocations processes should message [rc-help@colorado.edu](mailto:rc-help@colorado.edu) and [hpcsupport@cuanschutz.edu](mailto:hpcsupport@cuanschutz.edu), respectively."

Reply:
Do you know what the actual CSU email is?

Reply:
Is it rc2-request@colostate.edu ?